bin/util.js

```diff
@@ -181,12 +181,6 @@ exports.verify = async function(
   };
   const request = httpSigs.parseRequest(requestOptions, options);
   const signature = new Buffer(request.params.signature, 'base64');
-  if(!signature) {
-    throw new Error('No signature parameter found in Authorization header');
-  }
```

Comment:
I suspect this check wouldn't have worked anyway -- as signature would either be properly instantiated on the previous line or an exception would be thrown preventing this line from ever being hit.

Reply:
yeah sorry about this. the binary is pretty low quality. Any issues with the fix?

Reply:
No, I'll approve.